File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/While/Exercício 02.py
#Exercício - Média Idades

soma = 0
contador = 0

print("Digite as idades:")
idade = int(input(""))

while idade >= 0:

    soma = soma + idade
    contador = contador + 1

    idade = int(input(""))

if contador == 0:
    print("Impossível calcular!")
else:
    media = soma / contador
    print("A média das idades é = ", round(media,2))

File: 06 - C/Exercícios/03 - Estruturas Repetitivas/While/Exercício 02.c
//Exercício - Media Idades

#include <stdio.h>

int main(){

    int idade, soma, contador;
    double media;

    printf("Digite as idades:\n");
    scanf("%d", &idade);

    soma = 0;
    contador = 0;

    while(idade >= 0){
       
        soma = soma + idade;
        contador = contador + 1;

        scanf("%d", &idade);

    }

    if(contador == 0){
        printf("Impossivel calcular");
    }
    else{
        media = (double) soma / contador;
        printf("Media = %.2lf", media);
    }
    
    return 0;
}